pkg/kds/context/testdata/full_sync/mesh-only/global.golden.yaml

```diff
@@ -26,14 +26,14 @@ creationTime: "0001-01-01T00:00:00Z"
 healthCheck:
   time: "0001-01-01T00:00:00Z"
 modificationTime: "0001-01-01T00:00:00Z"
-name: zone-1
+name: zone-2
```

Comment:
this seems unrelated and is it causing the flakes we've seen?

Reply:
this is correct order and I'm working on this flake as we speak